### ISSUE TITLE
BUG: 'const' type qualifier on seccomp_api_get() return type has no effect

### ISSUE THREAD

**Issue by majek** [seccomp/libseccomp#111]
I like compiling with -Werror, and this popped up:

```
libseccomp/include/seccomp.h:302:1: error: 'const' type qualifier on return type has no effect [-Werror,-Wignored-qualifiers]
const unsigned int seccomp_api_get(void);
^~~~~~
1 error generated.
```

```
Ubuntu clang version 3.5.0-4ubuntu2~trusty2 (tags/RELEASE_350/final) (based on LLVM 3.5.0)
Target: x86_64-pc-linux-gnu
```

**Comment by pcmoore** [comment]
Thanks, I'll queue this up as something to fix in the v2.4 release.

**Comment by pcmoore** [comment]
This should be fixed in 3070d62f86a0cf7b62f5bd9024c695abab9dd877, thanks again for the bug report.
